### PR TITLE
Fix return error when secret webhook-receiver is not found

### PR DESCRIPTION
**Pull request by GGGitBoy**
**Problem:**

User has not configured alert that requires webhook-receiver, but will still receiver the error log `secret webhook-receiver is not found`.

**Solution:**

when the secret webhook-receiver cannot be found, it will not be output as an error

**Relate issue:**

https://github.com/rancher/rancher/issues/28954

### CODE REVIEW

**Comment by aiwantaozi** [comment]
If we enable MS team or other notifiers need this secret, and sever can't find this secret, and we directly return nil at this situation, it may cause problem.

**Review by aiwantaozi — COMMENTED** [review]
If we enable MS team or other notifiers need this secret, and sever can't find this secret, and we directly return nil at this situation, it may cause a problem.

**Review by orangedeng — APPROVED** [review]
LGTM

**Comment by orangedeng** [comment]
Please check the build failure.

**Comment by aiwantaozi** [comment]
Looks like this will only sync receivers when webhookReceiverEnabled, when the user only use notifiers no need to deploy webhookReceiver, the receivers won't update right?

File: pkg/controllers/managementuser/alert/configsyncer/configsyncer.go [hunk 1]
```diff
@@ -255,8 +255,10 @@ func (d *ConfigSyncer) sync() error {
 		logrus.Debug("The config stay the same, will not update the secret")
 	}
 
-	if err := d.syncReceiver(notifiers, cAlertGroupsMap, pAlertGroupsMap); err != nil {
-		return errors.Wrapf(err, "Update Webhook Receiver Config")
+	if webhookReceiverEnabled {
+		if err := d.syncReceiver(notifiers, cAlertGroupsMap, pAlertGroupsMap); err != nil {
```
Comment (aiwantaozi) [inline pkg/controllers/managementuser/alert/configsyncer/configsyncer.go:259]: This function is used for syncing configure for webhook-receiver, can we rename it?

**Review by aiwantaozi — APPROVED** [review]

**Review by aiwantaozi — APPROVED** [review]

**Review by dramich — APPROVED** [review]

**Review by aiyengar2 — COMMENTED** [review]
lgtm

**Pull request merged**